apps/server/src/modules/files-storage-client/service/copy-files.service.ts

```diff
@@ -1,67 +1,65 @@
 import { Injectable } from '@nestjs/common';
-import { EntityId, IComponentProperties, Lesson, Task } from '@shared/domain';
+import { CopyElementType, CopyHelperService, CopyStatus, CopyStatusEnum, EntityId } from '@shared/domain';
 import { CopyFileDto } from '../dto';
 import { EntityWithEmbeddedFiles } from '../interfaces';
 import { FileParamBuilder, CopyFilesOfParentParamBuilder } from '../mapper';
 import { FilesStorageClientAdapterService } from './files-storage-client.service';
 
 // TODO  missing FileCopyParams  ...passing user instead of userId
```

Comment:
see comment :D

Reply:
A: use of Generics has more advantages than using a locally defined type.
B: could not figure out the advantage